src/header/utils.test.js

```diff
@@ -28,4 +28,22 @@ describe('header utils', () => {
       expect(actualItems).toHaveLength(4);
     });
   });
+  describe('getToolsMenuItems', () => {
+    it('should include export tags option', () => {
+      setConfig({
+        ...getConfig(),
+        ENABLE_TAGGING_TAXONOMY_PAGES: 'true',
+      });
+      const actualItems = getToolsMenuItems(props);
+      expect(actualItems).toHaveLength(4);
```

Comment:
Could you please test that the items shown are the items we expect (instead of just checking the length)? It's ok to just check their titles.

Reply:
Sure! Done: effca983e2c8798eb00a1c67d4e510ee4bffcd31